bin/deploy.js

```diff
@@ -138,11 +138,18 @@ async function getInvalidationStatus(invalidationId, distribution) {
  */
 function _deployOptions(env) {
 	const conf = _loadConfigForNodeEnv(env);
-	return {
+
+	const deployOptions = {
```

Comment:
I'm definitely a fan of spaces over tabs, for the record...

Reply:
Not to keep harping on this but I thought we said tabs... Don't mind spaces though.

Reply:
Nah I'll stick with tabs. Just triple checking. Perhaps I will come around to it.